### DISCUSSION TITLE
Question: change loading-animation-icon

### DISCUSSION THREAD

**Discussion by aminamos** [muxinc/elements#637] [Q&A]
Hi, how can I change the loading-animation-icon to a custom one using the MUX Player (with next.js). do you have any example?

**Comment by dylanjha** [comment]
There's not currently a way to do this, but thanks for tracking here

**Comment by aminamos** [comment]
Reference: https://twitter.com/xperto_cms/status/1623435089970593792

**Comment by luwes** [comment]
This is possible now by using a custom theme but that might be too invasive.

An alternative is to hide the default media-loading-indicator with the new CSS var `--loading-indicator: none` in Mux player v0.19.0 and then use the player events like `waiting`, `playing`, etc to add a new loading indicator on top of the player.
